### DISCUSSION TITLE
Extensões bacanas para IDEs

### DISCUSSION THREAD

**Discussion by vitocaio** [frontendbr/forum#369] [Perguntas e Respostas]
Boa pessoal, quero saber se vocês tem dicas para extensões bacanas focadas no front end para algum desses carinhas ai, nome da extensão e para que serve.

Obrigado!

**Comment by matheusml** [comment]
Pro VS Code:
- Dracula Syntax Theme
- EditorConfig for VS Code
- ESLint

**Comment by vitocaio** [comment]
E qual é a utilidade destas ferramentas? @matheusml

**Comment by karlprieb** [comment]
Opa @vitocaio, vou responder por ele hehe :)

O Dracula é um tema para interface e do syntax highlight. Existe este tema pra vários editores, dá uma olhada https://draculatheme.com/

O EditorConfig é um plugin para configurar automaticamente o editor para os padrões de quebra de linha, espaçamento, etc do projeto. Leia mais em http://editorconfig.org/

O ESLint é um programa para verificar se o teu código javascript esta dentro dos padrões do projeto. Este é bem mais complexo por ter muito mais regras. Poder dar uma pesquisada nestas regras aqui https://eslint.org/docs/user-guide/configuring

**Comment by vitocaio** [comment]
Puts ai sim, vlw @karlprieb

**Comment by RatoX** [comment]
Eu sou do time do VIM.

Já venho configurando meu VIM pra front end tem um tempo, resolvi modularizar ele e disponibilizar no [github recentemente.](https://github.com/RatoX/vim-ratox)

**Comment by karlprieb** [comment]
Legal @RatoX ! Estou tentando usar VIM também :)
Porém optei pelo vimplug como plugin manager https://github.com/karlprieb/dotvim

**Comment by vitocaio** [comment]
Ainda não usei o Vim, ta valendo mais a pena o vim ou VS-Code?

**Comment by karlprieb** [comment]
@vitocaio Depende muito do teu workflow e de como tu deseja trabalhar.
